src/app/system/external-services/email/edit-email/edit-email.component.html

```diff
@@ -10,15 +10,20 @@
 
           <mat-form-field>
             <mat-label>Username</mat-label>
-            <input matInput required formControlName="username">
+            <input matInput type="email" required formControlName="username">
```

Comment:
username is not necessarily an email, so i dont think the type should be `email`

Reply:
Done removed!